zkevm-circuits/src/pi_circuit.rs

```diff
@@ -1018,6 +1116,255 @@ impl<F: Field> PiCircuitConfig<F> {
         Ok((offset + 1, data_hash_rlc_cell))
     }
 
+    /// Assign chunk txbytes, the pre-image to chunk_txbytes_hash
+    /// i.e. keccak256(rlc(chunk_txbytes)) == chunk_txbytes_hash.
+    fn assign_chunk_txbytes(
+        &self,
+        region: &mut Region<'_, F>,
+        offset: usize,
+        public_data: &PublicData,
+        tx_value_cells: &[AssignedCell<F, F>],
+        challenges: &Challenges<Value<F>>,
+    ) -> Result<(usize, AssignedCell<F, F>), Error> {
+        // Initialise the RLC accumulator and length values.
+        let (mut offset, mut rpi_rlc_acc, mut rpi_length) = self.assign_rlc_init(region, offset)?;
+
+        // Enable fixed columns for l2 tx hashRLC
+        for q_offset in
+            public_data.q_chunk_txbytes_start_offset()..public_data.q_chunk_txbytes_end_offset()
+        {
+            region.assign_fixed(
+                || "q_chunk_txbytes",
+                self.q_chunk_txbytes,
+                q_offset,
+                || Value::known(F::one()),
+            )?;
+        }
+
+        // Assign chunk txbytes
+        // Summary field TxHashRLC from TxTable is the RLC of a transaction's rlp_signed bytes.
+        // Its accumulation correctness is verified within the RLP Circuit and lookup-constrained
+        // from the TxCircuit.
+        let transactions = public_data
+            .transactions
+            .iter()
+            .filter(|&tx| tx.is_chunk_l2_tx())
```

Comment:
Here missed a tx_type byte. See the [comment](https://www.notion.so/PUBLIC-4844-Specifications-00bc5d9fcc244bd9a52bc59ac1523b04?d=5ef66bb6d0a1435497c602635cb44626&pvs=4#636b909d089a4490bd650742a196cc96)

Reply:
Make sense. Adding envelope byte to PI circuit input at once.